src/routes/test-contentful-content/+layout.svelte

```diff
@@ -0,0 +1,3 @@
+<div class="usa-prose grid-container">
```

Comment:
After looking at this a bit more I think we may also want to include the `usa-section` class to get the spacing right:
```suggestion
<div class="usa-section usa-prose grid-container">
```